docs/status.md

```diff
@@ -54,15 +54,15 @@ one of the following tracking labels.
 | complex                  |         no         |        yes*        |        yes*         |         yes          |        no        |
 | compute_reshape_shape    |         no         |         no         |         no          |         yes          |        no        |
 | concatenate              |         no         |        yes*        |        yes*         |         yes          |        no        |
-| constant                 |        yes         |        yes*        |        yes*         |         yes          |       yes        |
+| constant                 |        yes         |        yes         |         yes         |         yes          |       yes        |
 | convert                  |         no         |        yes*        |     infeasible      |         yes          |        no        |
 | convolution              |         no         |        yes*        |         no          |  yes(need-revisit)   |        no        |
-| cosine                   |        yes         |        yes*        |        yes*         |         yes          |       yes        |
+| cosine                   |        yes         |        yes         |         yes         |         yes          |       yes        |
 | create_token             |         no         |        yes*        |         no          |         yes          |        no        |
 | cross-replica-sum        |         no         |         no         |        yes*         |          no          |        no        |
 | cstr_reshapable          |         no         |        yes*        |         no          |         yes          |        no        |
 | custom_call              |         no         |        yes*        |     infeasible      |         yes          |        no        |
-| divide                   |        yes         |        yes*        |        yes*         |         yes          |        no        |
+| divide                   |        yes         |        yes         |         yes         |         yes          |        no        |
```

Comment:
StableHLO ODS allows boolean tensors in `divide` which doesn't make sense, while the spec and [shape_inference.cc](https://github.com/tensorflow/tensorflow/blob/c277b0b396b7709d4355d5794d380db90c22cc6a/tensorflow/compiler/xla/service/shape_inference.cc#L984-L999) don't. Looks like yet another ticket will be needed to align things. We're getting a lot of mileage out of this supposedly rountine PR! (Same for `remainder` and `subtract`).